CONTRIBUTING.md

```diff
@@ -12,6 +12,10 @@ Here's a list of tips and tricks that will help you get started.
 * TypeScript
 * WebIDL
 
+### Benchmarking
+
+Benchmarking helps prevent performance regression. 
```

Comment:
We might want to specify how to run them and show an example of what the output looks like

Reply:
Addressed first concern but planning to cleanup output in later PR and will document it there